src/table.tsx

```diff
@@ -407,6 +407,7 @@ export function Skeleton(props: React.PropsWithChildren & {readonly height?: num
 export function printTable<T extends Record<string, unknown>>(options: TableOptions<T>): void {
   const instance = render(<Table {...options} />)
   instance.unmount()
+  // It might be nice to have a "paddingBefore" and "paddingAfter" option for the number of newlines to enter.
```

Comment:
I don't think this is necessary. People can print as many newlines as they want before or after the table. This new line only exists because, otherwise, the next output will appear on the same line as the last line of the table. For example

```
┌───────┬─────────┬─────┬───────────────────────────────────────────────────────────────────────────┐
│  Id   │  Name   │ Age │                                Description                                │
├───────┼─────────┼─────┼───────────────────────────────────────────────────────────────────────────┤
│ 36329 │  Alice  │ 20  │  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod  │
│       │         │     │            tempor incididunt ut labore et dolore magna aliqua.            │
├───────┼─────────┼─────┼───────────────────────────────────────────────────────────────────────────┤
│ 49032 │   Bob   │ 21  │  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod  │
│       │         │     │            tempor incididunt ut labore et dolore magna aliqua.            │
├───────┼─────────┼─────┼───────────────────────────────────────────────────────────────────────────┤
│ 51786 │ Charlie │ 22  │  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod  │
│       │         │     │            tempor incididunt ut labore et dolore magna aliqua.            │
└───────┴─────────┴─────┴───────────────────────────────────────────────────────────────────────────┘my next line
```